nftables.go

```diff
@@ -58,29 +65,42 @@ func (n *nft) Init() error {
 	}
 	n.set = set
 
-	n.conn.AddRule(&nftables.Rule{
+	r := &nftables.Rule{
 		Table: n.table,
 		Chain: chain,
-		Exprs: []expr.Any{
-			// [ payload load 4b @ network header + 16 => reg 1 ]
-			&expr.Payload{
-				DestRegister: 1,
-				Base:         expr.PayloadBaseNetworkHeader,
-				Offset:       12,
-				Len:          4,
-			},
-			// [ lookup reg 1 set whitelist ]
-			&expr.Lookup{
-				SourceRegister: 1,
-				SetName:        n.set.Name,
-				SetID:          n.set.ID,
-			},
-			//[ immediate reg 0 drop ]
-			&expr.Verdict{
-				Kind: expr.VerdictDrop,
-			},
-		},
+		Exprs: []expr.Any{},
+	}
+	// [ payload load 4b @ network header + 16 => reg 1 ]
+	r.Exprs = append(r.Exprs, &expr.Payload{
+		DestRegister: 1,
+		Base:         expr.PayloadBaseNetworkHeader,
+		Offset:       12,
+		Len:          4,
 	})
+	// [ lookup reg 1 set whitelist ]
+	r.Exprs = append(r.Exprs, &expr.Lookup{
+		SourceRegister: 1,
+		SetName:        n.set.Name,
+		SetID:          n.set.ID,
+	})
+	if n.DenyLog {
+		r.Exprs = append(r.Exprs, &expr.Log{
+			Key:  unix.NFTA_LOG_PREFIX,
+			Data: []byte(n.DenyLogPrefix),
+		})
+	}
+	if n.DenyAction == "reject" {
```

Comment:
beware, it was upper-cased earlier, so it should be REJECT if I'm not mistaken

Reply:
updated

Reply:
Hi, does it look good now with the update?

Reply:
Hey, sorry for delay, having a look at it now :)